sites/docs/usage/env.rst

```diff
@@ -166,6 +166,18 @@ to act as if they have been called with ``pty=False``.
 .. seealso:: :option:`--no-pty`
 .. versionadded:: 1.0
 
+.. _banner_timeout:
+
+``banner_timeout``
+------------------
+
+**Default:** ``15``
+
+SSH banner timeout, in seconds.
+
+.. versionadded:: 1.9
```

Comment:
This should be `.. versionadded:: 1.11` since `1.10.x` is the latest stable release.